herokuappTests/auth/authenticator.py

```diff
@@ -0,0 +1,10 @@
+import requests
+
+class Authenticator:
+    def get_token(self):
+        payload = {
```

Comment:
Don't hardcode these values inside code itself. 
Create class Constants in constants.py file where you will keep all your data stuff including urls/usernames/passwords, some shared test data etc. 
Also, make sure that some of these variables can be provided externally by exporting variables before running test.
For example:
`export BASE_URL=https://restful-booker.herokuapp.com`

Also, keep in mind that endpoints like /auth here shouldn't be part of this constant